lib/jirawatch/cli/track.rb

```diff
@@ -55,6 +86,14 @@ def call(issue_key:, **options)
           puts "Worklog was empty, time was not tracked" if worklog_lines.empty?
 
           File.delete worklog_file
+          exit
+        end
+
+        loop do
+          reader.read_line("")
+        rescue Interrupt
+          # This is executed when the user presses CTRL-C, the only thing to do is to quit the program
```

Comment:
What about CTRL-P to pause/restart? Because when the user presses the `space` bar a space is printed in the terminal and the next showed message is shifted to the right like that: `   message`

Reply:
Yeah that might be better, it would also prevent accidental pauses too